packages/ui/src/components/molecules/CardProfile.tsx

```diff
@@ -108,17 +106,19 @@ const CenteredCounter = styled(FlexBox)`
   }
 `
 
+const LogoutIconStyled = styled(Icon)``

```

Comment:
If you don't apply any custom styles, does the simple icon not work?

Reply:
I will check, I just automatically changed the names and didn't think on it...